packages/typespec-ts/src/modular/buildCodeModel.ts

```diff
@@ -267,10 +276,11 @@ function getEffectiveSchemaType(program: Program, type: Model | Union): Model {
 function processModelProperties(
   context: SdkContext,
   newValue: any,
-  model: Model
+  model: Model,
+  usage: UsageFlags
```

Comment:
I think we should add usage here and in the other places where it is being added for the first time, as part of an options bag

Reply:
But if there is no obvious dfault value for this and has to be required maybe it makes sense to keep it as positional